src/cljdoc/render/build_log.clj

```diff
@@ -123,7 +123,10 @@
    (section
     (:api_imported_ts build-info)
     [:h3.mt0 "API Import"]
-    [:p.bg-washed-green.pa3.br2.ma0 "API imported successfully"]
+    (if (or (nil? (:has_namespaces build-info))
```

Comment:
At this point `:has_namespaces` should always be set so let's maybe do something like
```
(some-> build-info :has_namespaces zero?)
```
the `some->` will achieve that `zero?` is only called if the `:has_namespaces` field is actually set.

Reply:
Or `(some-> build-info :imported_namespaces zero?)` if we decide to store the number of imported namespaces.